examples/ui/.env.e2e

```
@@ -1 +1,3 @@
 NEXT_PUBLIC_E2E=true
+NEXT_PUBLIC_ANVIL_URL=http://127.0.0.1:8545
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_ANVIL_URL key should go before the NEXT_PUBLIC_E2E key

(UnorderedKey)